Swift/BRCrypto/BRCryptoWalletManager.swift

```diff
@@ -931,7 +931,69 @@ extension WalletManager {
 
 extension WalletManager {
     internal var clientGEN: BRCryptoCWMClientGEN {
-        return BRCryptoCWMClientGEN ()
+        return BRCryptoCWMClientGEN (
+            funcGetBlockNumber: { (context, cwm, sid) in
+                let manager = Unmanaged<WalletManager>.fromOpaque(context!).takeUnretainedValue()
+                precondition (nil != cwm, "SYS: GEN: GetBlockNumber: Missed {gwm}")
+
+                print ("SYS: GEN: GetBlockNumber")
+                manager.query.getBlockchain (blockchainId: manager.network.uids) {
+                    (res: Result<BlockChainDB.Model.Blockchain, BlockChainDB.QueryError>) in
+                    res.resolve (
+                        success: { cwmAnnounceGetBlockNumberSuccessAsInteger (cwm, sid, $0.blockHeight) },
+                        failure: { (_) in cwmAnnounceGetBlockNumberFailure (cwm, sid) })
+                }},
+
+            funcGetTransactions: { (context, cwm, sid, address, begBlockNumber, endBlockNumber) in
```

Comment:
As a follow up to track (i.e. not blocking the PR), we might want address to be a list of addresses.

Reply:
CORE-415